.vscode/launch.json

```diff
@@ -92,6 +92,30 @@
 			"webRoot": "${workspaceFolder}",
 			"timeout": 45000
 		},
+		{
```

Comment:
this is for your local testing only, I don't know whether it's worth adding it here.

Reply:
i think it's beneficial to have at least one scenario here that exercises command line code. it will help get coverage via usage.

Reply:
Can we make it more generic instead of being specific to one command?